### ISSUE TITLE
Nerfed EXP drops

### ISSUE THREAD

**Issue by mrcoffee1026** [Nathat23/StackMob-5#80]
This isn't dropping the appropriate EXP per mob.  The defaults are set up to nerf drops by default since the minimum "multiplier" is 0.5, assuming the "multiplier" is indeed a number being multiplied by the number of mobs and then the amount of exp one would have dropped, but that is not what is actually going on here... I can't conjure up a formula that would produce the actual results.

Endermen are a good example since they always drop exactly 5.0 exp.  My stack sizes are a minimum of 10, a maximum of 25, therefore killing a stack should produce no less than 50 exp or 2625 exp max if I set the multiplier minimum to 1.0, which I have, but I get anywhere between 11-20 exp per stack.  Maybe you've set every mob to a static 1.0 exp or less per mob in a stack to create this issue, I'm not sure, but there should be some method of recreating vanilla-level exp drops here and without making dramatic changes to the multiple which would make killing a stack of sheep as op as killing a stack of endermen, there doesn't appear to be a way to do so at this time.  Please fix the math in this formula?

Stackmob 5.1.1

**Comment by Zephyure** [comment]
Same problem for me, however i believe that the stacking for xp is also a problem since i have a guardian farm and it drops the right amount of drops, but i cannot stop the experience orbs from stacking meaning it doesn't mend items with the mending enchant, also killing 100 guardians gives much more experience than it currently gives. If there is a way to stop the xp from stacking please tell me.

**Comment by Nathat23** [comment]
A new build can be downloaded from https://ci.ecocitycraft.com/job/StackMob-5/
Please let me know if this needs to be tweaked further.

**Comment by mrcoffee1026** [comment]
Yeah that seems to be doing the trick.  First stack i tried gave 141.  Max I put in the original post is wrong should have said 125 max... but my multiplier min is 1.0 and max 2.0, so any amount of exp between 50 and 250 should be possible.  Since I got 141 on the first stack I tried, this seems to be true now... in my grinder the counts are difficult to read but they do always seem to be falling within that expected range now, thank you very much!
